scripts/interleave-reads.py

```diff
@@ -59,11 +59,12 @@ def get_parser():
     parser.add_argument('infiles', nargs='+')
     parser.add_argument('-o', '--output', metavar="filename",
                         type=argparse.FileType('w'),
-                        default=sys.stdout)
+                        default=None)
```

Comment:
It messes with detecting if the user gave a name--normally we'd just check to see if `args.output is not none` or summuch, but if we make it standard to sys.stdout it always looks like `-o` has been specified.

In interleave- if we don't specify `-o` the script automatically comes up with its own names.